Dart/thirdPartySrc/analysisServer/com/google/dart/server/internal/remote/RemoteAnalysisServerImpl.java

```diff
@@ -715,6 +744,59 @@ else if (event.equals(LAUNCH_DATA_NOTIFICATION_RESULTS)) {
     return true;
   }
 
+  /**
+   * Attempts to handle the given {@link JsonObject} as a request. Return {@code true} if it
+   * was handled, otherwise {@code false} is returned.
+   *
+   * @return {@code true} if it was handled, otherwise {@code false} is returned
+   */
+  private boolean processRequests(final JsonObject response, final String idString) throws Exception {
```

Comment:
So, I rewrote `processRequests()` so that it calls `server_openUrlRequest` or `server_showMessageRequest` as needed